src/main/java/edu/unc/lib/boxc/migration/cdm/services/RedirectMappingIndexService.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.migration.cdm.services;
+
+import edu.unc.lib.boxc.migration.cdm.exceptions.InvalidProjectStateException;
+import edu.unc.lib.boxc.migration.cdm.exceptions.MigrationException;
+import edu.unc.lib.boxc.migration.cdm.model.MigrationProject;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.DriverManager;
+
+/**
+ * Service for indexing the redirect mapping CSV into the chomping_block DB
+ *
+ * @author snluong
+ */
+public class RedirectMappingIndexService {
+    private final MigrationProject project;
+
+    public RedirectMappingIndexService(MigrationProject project) {
+        this.project = project;
+    }
+
+    public void indexMapping() {
```

Comment:
I know it's not going to say a lot, but please add javadoc to this method to briefly describe what the method does. Typically we want to have javadoc for all public methods that aren't just boilerplate like getters and setters.

Reply:
ok!